src/index.js

```diff
@@ -60,20 +60,54 @@ export default function boolean(subject, clipping, operation) {
   const sbbox = [Infinity, Infinity, -Infinity, -Infinity];
   const cbbox = [Infinity, Infinity, -Infinity, -Infinity];
 
-  //console.time('fill queue');
+  // console.time('fill queue');
   const eventQueue = fillQueue(subject, clipping, sbbox, cbbox, operation);
   //console.timeEnd('fill queue');
 
   trivial = compareBBoxes(subject, clipping, sbbox, cbbox, operation);
   if (trivial) {
     return trivial === EMPTY ? null : trivial;
   }
-  //console.time('subdivide edges');
+  // console.time('subdivide edges');
   const sortedEvents = subdivideSegments(eventQueue, subject, clipping, sbbox, cbbox, operation);
   //console.timeEnd('subdivide edges');
 
-  //console.time('connect vertices');
+  // console.time('connect vertices');
   const result = connectEdges(sortedEvents, operation);
   //console.timeEnd('connect vertices');
-  return result;
+
+  const out = [];
+  for (var i = 0; i < result.length; i++) {
+    const contour = result[i];
+    const points = contour.points;
+    if (contour.external) {
+      if (points[0][0] !== points[points.length - 1][0] ||
+          points[0][1] !== points[points.length - 1][1]
+      ) {
+        points.push([points[0][0], points[0][1]]);
+      }
+
+      const outCoords = [];
```

Comment:
This line doesn't need to be in the other scope, could be moved into line 93 to make it `const outCoords = [contour.points]`.

Reply:
true